pkg/cli/cmds/k3sopts.go

```diff
@@ -9,13 +9,14 @@ import (
 )
 
 var (
-	Drop = &K3SFlagOption{
+	drop = &K3SFlagOption{
 		Drop: true,
 	}
-	Hide = &K3SFlagOption{
+	hide = &K3SFlagOption{
 		Hide: true,
 	}
 )
+var copy *K3SFlagOption = nil
```

Comment:
@briandowns I appreciate what you've done with this change although some call-out as to why you are moving to package scope in a PR comment might be helpful. Also, this change, overall, makes the PR much noisier and harder to asses re: "secrets-encryption flag".

Reply:
Understood. I'll spit this out in the future. As for the "why", it's not used outside of the package.